contracts/atm/ATMGovernance.sol

```diff
@@ -54,6 +58,7 @@ contract ATMGovernance is SignerRole, IATMGovernance {
     function addGeneralSetting(bytes32 settingName, uint256 settingValue)
         external
         onlySigner()
+    // TODO Do we need to add isInitialized() (the same for other functions)?
```

Comment:
I think that would be a good idea as a general check, but not entirely necessary